src/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+  <form class="form form-big"  data-qa="big">
+    <input class="search-input search-input-big"  type="text"  placeholder="Try &#34;Los Angeles&#34;">
```

Comment:
цей input елемент блоку форми тому потрібно записувати таким чином
```suggestion
    <input class="search-form__input search-form__input_big"  type="text"  placeholder="Try &#34;Los Angeles&#34;">
```

Reply:
ok